bot/resources/evergreen/game_recs/chrono_trigger.json

```diff
@@ -0,0 +1,7 @@
+{
+  "title": "Chrono Trigger",
+  "recText": "One of the best games of all time. A brilliant story involving time-travel with loveable characters. It has a brilliant score by Yasonuri Mitsuda and artwork by Akira Toriyama. With over 20 endings and New Game+, there is a huge amount of replay value here.",
+  "wikiLink": "https://rawg.io/games/chrono-trigger-1995",
```

Comment:
Just `link` would suffice for this field. rawg.io isn't really a wiki anyway. Also, let's avoid camel case.